src/applications/vaos/tests/mocks/setup.js

```diff
@@ -109,6 +110,42 @@ export function renderFromRoutes({ initialState, store = null, path = '/' }) {
   return { ...screen, history };
 }
 
+/*
+ * This function returns a date for which adjusting the timezone
+ * to the provided zone results in a date on a different day.
+ * 
+ * For example, if you're on ET and you call this function with 
+ * America/Denver, then you'll get a time of 12:30 am, because that will
+ * be a different day of the month than the same time in America/Denver
+ * 
+ * If the local zone and the passed zone are the same, you'll get a 12:30
+ * am date, similar to zones that are ahead of the passed zone.
+ * 
+ * @export
+ * @param {string} zone A timezone description
+ * @returns An ISO date string for a date that will cross over midnight
+ */
+export function getTimezoneTestDate(zone = 'America/Denver') {
```

Comment:
My hope with this is that it surfaces those failed tests we see just after midnight UTC locally and in Jenkins at any time of the day.

Reply:
Great idea adding this along with MockDate.  It should definitely alleviate the issues we're having with flaky tests if we use it consistently.